### PR TITLE
Release 0.3.1: Dataflow labels

### PR DESCRIPTION
**Pull request by peel**
A quick and small release to roll out #42 needed for beta deployment.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Version 0.3.1 (2020-03-??)
```
Comment (dilyand) [inline CHANGELOG]: Shouldn't it be `0.4.0` since we're adding a feature, even though it's small?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+--------------------------
+Add support for GCP labels in dataflow launcher docker image (#42)
+Bump scio to 0.8.1 (#41)
```
Comment (dilyand) [inline CHANGELOG]:
```suggestion
Bump scio to 0.8.2 (#41)
```

File: CHANGELOG (hunk 1 above)
Comment (peel) [reply CHANGELOG]: I'm not sure. I would agree for the most part, but on the other hand it's a compatibility-release so I'd stick to 0.3 series for this one.

Comment (dilyand) [reply CHANGELOG]: According to semver, patches are just for bugfixes and minor version changes are for backwards compatible feature additions.

But I'm just being a pita. It don't really think it's very important.

**Pull request merged**